docs/topics/articles.md

```diff
@@ -0,0 +1,184 @@
+# `Article`s and `ListingPage`s
+
+`Article`s are publishable content types that live in collections. Normally
+articles are mounted under a parent's URL.
+
+An example of an article is a Press Release, which would be mounted under a
+Press Release Listing Page. The Listing Page has the URL "/press" and thus all
+the Press Releases have the URL "/press/<slug>".
+
+## `ArticleBase`
+
+The `icekit.articles.models.ArticleBase` inherits `title`, `slug` and
+publishing fields, and each `ArticleBase` subclass implements some key
+functionality:
+
+1. **Links to parents.**
+In order for an Article to know its URL (which is based on a parent's URL), it
+should define a `parent` link, which is normally a link to a `ListingPage`.
```

Comment:
This explanation is a little clearer to me. Thanks.

Still, `parent` is a very generic name to begin with and is also overloaded and has different implementations. If we're suggesting that `parent` might be an FK to `UrlNode` or to something else like `Category` (for a more typical implementation), we should say that in the docs. It sounded to me like it would always be a page (`UrlNode`).

If the only reason we're using the name `parent` is to re-use templates, maybe it's worth changing the name and using different templates?

Could we make the implementation consistent by insisting that parent must be an FK? I can imagine that this could easily cause us problems and inefficient queries.

Finally, are we doing anything to make the page selection easier? E.g. showing a model choice field that is filtered to show only `FooListingPage` instances? I can imagine that a popup version of the entire page tree change list page would not be very friendly to use, and that allowing any type of `UrlNode` instance to be linked as the parent for an article could be problematic.